blocks/core/ff_module/ff_module-task-event/src/TaskEventContainer.rt

```diff
@@ -0,0 +1,6 @@
+<div
```

Comment:
I realised (somewhat belatedly) that "container" was a bad choice of name on my part since there's already an unrelated notion of container components in React. I struggled to think of an appropriate alternative name for `TaskEventContainer` though.